lib/generators/nocms/blocks/layout_generator.rb

```diff
@@ -13,6 +13,7 @@ def generate_stylesheets
       def generate_views
         template "app/views/no_cms/blocks/blocks/layout.html.erb", File.join(destination_root, "app/views/no_cms/blocks/blocks", "_#{name}.html.erb")
         template "app/views/no_cms/admin/blocks/blocks/layout.html.erb", File.join(destination_root, "app/views/no_cms/admin/blocks/blocks", "_#{name}.html.erb")
+        template "app/views/no_cms/blocks/skeletons/default.html.erb", File.join(destination_root, "app/views/no_cms/blocks/skeletons", "_#{name}.html.erb")
```

Comment:
I think someone forgot to commit a file, because this is throwing an error on Circle CI: `Could not find "app/views/no_cms/blocks/skeletons/default.html.erb" in any of your source paths. Your current source paths are: 
/home/ubuntu/nocms-blocks/lib/generators/nocms/blocks/templates`